pkg/handlers/clusterlogging/accesscontrol/documentmanager_test.go

```diff
@@ -0,0 +1,107 @@
+package accesscontrol
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
what does the proceeding `.` do?

Reply:
it pulls the functions into the global namespace so you don't have to specify the package when using them.  This is the pattern from the documentation